templates/deployment.yaml

```diff
@@ -250,8 +255,12 @@ spec:
           value: "http://$(JAEGER_AGENT_HOST):5778/sampling"
         - name: "OFFLOAD_TO_HOST"
           value: "127.0.0.1"
+        - name: "OFFLOAD_TO_HOST_GRPC"
```

Comment:
This and `OFFLOAD_TO_PORT_GRPC` are two new env vars introduced in the new version of `openresty-sidecar`.